index2.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+   
+    <link rel="stylesheet" href="styles2.css">
+</head>
+<body>
+    <div id="smallScreen">
+        <span><a href=""></a>Welcome</span>
+        <span><a href=""><i class="fa fa-align-justify"></i></a></span>
+    </div>
+    <div id="main">
+        <div class="navigation">
+            <nav><ul>
+                <li><a href="#">Portoflio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul></nav>
+        </div>
+        <div class="headline">
```

Comment:
- [ ] What the previous reviewer wanted to mean here is to change the div on line 36 into a section tag, it is recommended to use them when comes to the main container for the best semantic approach instead of a simple div, please update that
- [ ] [OPTIONAL] In addition, you may want to change the header tag on line 37 into a div, you are doing fine by using section tags but not in a good way and places, please read this [article](https://www.thoughtco.com/html5-section-element-3467994) for a better understanding